skein/cli.py

```diff
@@ -139,6 +139,8 @@ def application_client_from_app_id(app_id):
           help="If provided, the driver will write logs here.")
 log_level = arg("--log-level", default=None,
                 help="The driver log level, default is INFO")
+java_opts = arg("--java_opts", default=None, action='append',
+                help="Additional options for the driver")
```

Comment:
Slight preference for `java-option` rather than `java-opt`, the abbreviation only saves 3 characters.  Also, preference for `-` over `_` when joining words. 

If we're using an append option, perhaps it should be singular (`java-option`) rather than plural (`java-options`), to better signal that the flag only takes a single option?

```bash
skein driver start --java-option -Dsome.option.one=foo --java-option -Dsome.option.two=bar
# vs
skein driver start --java-options -Dsome.option.one=foo --java-options -Dsome.option.two=bar
```

Reply:
Alternatively, could do this with an environment variable? I'm not sure what's common practice here.

```
SKEIN_DRIVER_JAVA_OPTIONS="-Dsome.option.one=foo -Dsome.option.two=bar"
```

Reply:
OK. Thanks. Will do the changes. I prefer injecting the options. Temporary I already have fixed it by adding this stuff to JAVA_TOOL_OPTIONS. It will be picked up by the jvm.

We actually have a shared gateway with 48 cores to launch our tensorflow jobs with skein on the hadoop cluster. Many people launch multiple jobs at the same time.
The java8 defaults for the GC are not very well, by default it creates 66% * number of available cores. So on a big machine this instantiates a lot of threads and as by default one job creates one skein daemon it will allocate a lot of resources.
So the first fix is to create one single daemon for multiple skein jobs for each user (which is already possible). And the second fix is to tune the allocations of the skein java daemon i.e. by reducing the number of threads for the GC (ParallelGCThreads=1)

Reply:
Done. I changed it as requested